code/src/components/Form.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+ 
+
+const Form = ({newMessage, onNewMessage, onFormSubmit}) => {
+
+    // if (newMessage.length > 10) {
+    //     return ( <p>Your message is too long</p>)
+    // }
+    
+
+return(
+
+        <form id="form" 
+        onSubmit={onFormSubmit}>
+
+            <div className="input-group">
+                <label htmlFor="newMessage" key="newMessage">What's making you happy right now?
+
+                    <textarea type="text" 
+                    name="newMessage" 
+                    value={newMessage}
+                    className="text-input" 
+                    onChange={onNewMessage}
+                    
+                    />
+                    
+                </label>
+
+            </div>
+
+
+            <div className="btn-group">
+                <button 
+                disabled={newMessage.length < 5}
+                type="submit"
+                className="send-btn"> 
+                    <span role="img" aria-label="heart emoji">❤️ </span> Send Happy Thought <span role="img" aria-label="heart emoji"> ❤️</span>
+                </button>
+
+                <span 
+                className={newMessage.length > 140 ? "characters red-text" : "characters"}>
```

Comment:
Good that u can se how many letters u have left to use, and that u also se when u have minimum 6 that the button change color